### PR TITLE
feature/176717631-access_post_from_blogpage

### PR DESCRIPTION
**Pull request by marijana-kolosek**
screenrecording:
https://www.awesomescreenshot.com/video/2675164?key=9671ba3b4344dcfee8c4bffe3ac493e2

As a User, I should be able to access the post from the blog page:
https://www.pivotaltracker.com/story/show/176717631

### CODE REVIEW

File: src/components/AuthorsPage.jsx [hunk 1]
```diff
@@ -45,4 +38,6 @@ const AuthorsPage = (props) => {
   );
 };
 
-export { AuthorsPage };
+const mapStateToProps = (state) => ({ authors: state.authors.authors });
```
Comment (salac-kolosek) [inline src/components/AuthorsPage.jsx]: @marijana-kolosek Ovo mozes i preko hook-ova.

File: src/components/AuthorsPage.jsx [hunk 2]
```diff
@@ -45,4 +38,6 @@ const AuthorsPage = (props) => {
...
+
+export default connect(mapStateToProps, { fetchAuthors })(AuthorsPage);
```
Comment (salac-kolosek) [inline src/components/AuthorsPage.jsx]: @marijana-kolosek Isto preko hook-a.

**Review by salac-kolosek — APPROVED** [review]
Dobro je samo sam ostavio malu napomenu.

File: src/components/AuthorsPage.jsx (hunk 1 above)
Comment (marijana-kolosek) [reply src/components/AuthorsPage.jsx]: mislite na useSelector() ?

**Review by salac-kolosek — APPROVED** [review]

File: src/components/AuthorsPage.jsx (hunk 1 above)
Comment (salac-kolosek) [reply src/components/AuthorsPage.jsx]: @marijana-kolosek  Da.

**Pull request merged**